optimum/exporters/tflite/config.py

```diff
@@ -31,3 +31,11 @@ class TextEncoderTFliteConfig(TFLiteConfig):
 
     DUMMY_INPUT_GENERATOR_CLASSES = (DummyTextInputGenerator,)
     MANDATORY_AXES = ("batch_size", "sequence_length", ("multiple-choice", "num_choices"))
+
+
+class VisionTFLiteConfig(TFLiteConfig):
+    """
+    Handles vision architectures.
+    """
+
+    DUMMY_INPUT_GENERATOR_CLASSES = (DummyTextInputGenerator,)
```

Comment:
After debugging it, I found it was due to the fact that you specified the wrong dummy input generator. 
```suggestion
    DUMMY_INPUT_GENERATOR_CLASSES = (DummyVisionInputGenerator,)
```
This should solve your issue

Reply:
Also you need to specify the MANDATORY_AXES class attribute in `VisionTFLiteConfig`:

```python
MANDATORY_AXES = ("batch_size", "num_channels", "width", "height")
```